src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,21 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+
+function TodoList(props){
+    return (
+        <div className='todo-list'>
+            {props.todos.map(todo =>
+                <Todo
+                    key={todo.task}
+                    todo={todo}
+                    />
+                )}
```

Comment:
The indentation here if off a bit; don't forget to use [prettier](https://prettier.io) for prettier code!